pkgs/development/python-modules/wxPython/4.0.nix

```diff
@@ -26,8 +29,16 @@ buildPythonPackage rec {
 
   doCheck = false;
 
-  nativeBuildInputs = [ pkgconfig which doxygen wxGTK ];
-  buildInputs = [ ncurses wxGTK.gtk ];
+  nativeBuildInputs = [ pkgconfig which doxygen ]
+  ++ (if !stdenv.isDarwin then [ wxGTK ] else [ wxmac ]);
```

Comment:
And if I want to use wxwidgets on macOS?

Why not only wxwidgets as dependency and in callPackage you can do:
```
wxwidgets = stdenv.isDarwin ? wxmac : wxGTK
```

otherwise I have to employ an overide/overlay like `wxmac = wxwidgets`, which seems odd to me and I could imagine some users not having the idea that they could do that.

Reply:
@hannesweisbach I agree with your comment. It will, however, require a more sophisticated refactoring (e.g. to build with `wxGTK.gtk` and perhaps exclude the frameworks).

Reply:
I'm not sure I follow ;) I just wanted to bring up that wxGTK build on macOS too, and that wxmac is not necessarily required.